gui/tooltips.lua

```diff
@@ -0,0 +1,446 @@
+-- Show tooltips on units and/or mouse
+
+--@ module = true
+
+local RELOAD = false -- set to true when actively working on this script
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local overlay = require('plugins.overlay')
+local ResizingPanel = require('gui.widgets.containers.resizing_panel')
+
+--------------------------------------------------------------------------------
+
+-- pens are the same as gui/control-panel.lua
+local textures = require('gui.textures')
+local function get_icon_pens()
```

Comment:
this is ugly duplication, but we don't have a good widget API for it yet, so I don't think there's a better option. We can change this later once we have a better way to address this fairly common use case. I see `orders.lua` does some similar duplication for the workshop labor restrictions overlay.